.drone.yml

```diff
@@ -55,6 +55,13 @@ steps:
       repo: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator
       auto_tag: true
 
+  - name: validate-operator-imgage
+    image: quay.io/${DRONE_REPO_NAMESPACE}/performance-addon-operator:${DRONE_TAG}
+    commands:
+      - ls /usr/local/bin/ | grep performance-operator
```

Comment:
maybe do directly `ls /usr/local/bin/performance-operator` ? if this fails it's already a bad thing

Reply:
TIL you can `ls` a file :slightly_smiling_face: I can't remember doing that before, thanks!